README.md

```diff
@@ -53,6 +53,101 @@ Add this to your project's `pom.xml` file:
 ```
 
 
+## Getting Started
+
+### Add your Android Platform
+To initialize your SDK and start interacting with Appwrite services, you need to add a new Android platform to your project. To add a new platform, go to your Appwrite console, choose the project you created in the step before, and click the 'Add Platform' button.
+
+From the options, choose to add a new **Android** platform and add your app credentials.
+
+Add your app <u>name</u> and <u>package name</u>. Your package name is generally the applicationId in your app-level `build.gradle` file. By registering a new platform, you are allowing your app to communicate with the Appwrite API.
+
+### OAuth
```

Comment:
Any idea for a different title? we might use the activity setting for more stuff in the future beside OAuth.

Reply:
I have updated the title. Do you think its better ?